tidy3d/components/geometry/utils_2d.py

```diff
@@ -63,17 +59,37 @@ def set_bounds(geom: Geometry, bounds: Tuple[float, float], axis: Axis) -> Geome
         new_center[axis] = (bounds[0] + bounds[1]) / 2
         new_length = bounds[1] - bounds[0]
         return geom.updated_copy(center=new_center, length=new_length)
+    if isinstance(geom, Transformed):
+        min_bound = np.array([0, 0, 0, 1.0])
```

Comment:
out of curiosity, why is this a z unit vector? should it be set to the unit normal vector for the geometry?

Reply:
That is a good question. I think I got the idea from a different [section](https://github.com/flexcompute/tidy3d/blob/844b6bab56232190e531cadec36de22d156cf5ca/tidy3d/components/geometry/base.py#L2297-L2320).

They arrays are actually made to be size 4, so that the Transform can be applied. I think I need to double-check this part though.

Reply:
Yea I think the 1 is needed to apply translation transformations, but maybe @lucas-flexcompute can just confirm that it is ok?

Reply:
Yes, the 1 at the 4th position is to set up [homogeneous coordinates](https://en.wikipedia.org/wiki/Homogeneous_coordinates) for 3D transformations.